src/main/java/org/jenkinsci/plugins/workflow/cps/CpsThreadGroup.java

```diff
@@ -420,6 +420,9 @@ void saveProgram() throws IOException {
 
     @CpsVmThreadOnly
     public void saveProgram(File f) throws IOException {
+        boolean logging = LOGGER.isLoggable(Level.FINER);
```

Comment:
Could. Anyway I plan to revert this and incorporate it into a more general timing reporting system I am working on for Pipeline builds.